db/migrate/20230220234054_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] Great job for adding the columns here in  posts.rb but I think you forget to add is user table `commentss_counter`  & `likes_counter` as shown in the ERD diagram so kindly add a new migration and add this column 🤩💯⭐✨

## Kindly Use `snake_case` for methods and variables: 
### You can do somthing like this ⬇💪🏻 This will add a new migration for you!
> rails generate migration AddPostsCounterToUsers

### And now inside your migration add a new column 💯💯⭐
> add_column :posts, :comments_counter, :integer
> add_column :posts, :likes_counter, :integer